packages/ord-connect/src/components/SelectWalletModal/index.tsx

```diff
@@ -72,6 +75,71 @@ export function SelectWalletModal({
     [disconnectWallet],
   );
 
+  const onConnectMagicEdenWallet = useCallback(async () => {
+    if (network === "testnet") {
+      const unsupportedNetworkError = new Error(
+        "Magic Eden wallet is not supported on testnet",
+      );
+      onError(Wallet.MAGICEDEN, unsupportedNetworkError);
+      return false;
+    }
+
+    try {
+      setErrorMessage("");
+      const magicEden = await getMagicEdenAddress(network);
```

Comment:
some simple consistency on the variables naming to ensure readers know what the variable holds
```suggestion
      const magicEdenAddresses = await getMagicEdenAddress(network);
```